crates/optimism/node/src/node.rs

```diff
@@ -205,6 +206,7 @@ where
 
 impl NodeTypes for OpNode {
     type Primitives = OpPrimitives;
+    type Primitives = EthNetworkPrimitives;
```

Comment:
this should probably be
```suggestion
    type NetworkPrimitives = EthNetworkPrimitives;
```
I think

Reply:
whoops yeah